reference/6/Microsoft.PowerShell.Core/Enter-PSSession.md

```diff
@@ -629,6 +629,30 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Subsystem
+Specifies the SSH subsystem used for the new **PSSession**.
+
+This specifies the subsystem to use on the target as defined in sshd_config.
+
+The session configuration for a session is located on the remote computer.
```

Comment:
We should not conflate "subsystem" and "session configuration", since they are two entirely different things.  $PSSessionConfigurationName should never specify a powershell subsystem.  A subsystem entry just specifies a pwsh.exe version to use as the SSH transport endpoint host.  The session configuration of any SSH endpoint is currently fixed to the default session configuration.  At some point in the future we will add support for SSH custom session configuration and then $PSSessionConfigurationName will be applicable.

Reply:
I think I cut and pasted this from the `-ConfigurationName` parameter.  Will clean up.